package.json

```diff
@@ -6,12 +6,12 @@
     "clean": "rimraf packages/**/lib"
   },
   "devDependencies": {
-    "@types/glob": "5.0.35",
+    "@types/glob": "7.2.0",
     "@types/gulp": "4.0.5",
     "@types/jest": "24.0.15",
     "@types/node": "6.0.111",
-    "@types/ramda": "0.25.29",
-    "@types/uuid": "3.4.3",
+    "@types/ramda": "0.27.64",
```

Comment:
This is not a dev dependency. This library is used as a way to bring functional programming to JS. Are we actually using it? Maybe it can be removed..

Reply:
I am planning to remove it, in the new versions